backend/main.go

```diff
@@ -0,0 +1,480 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"os/signal"
+	"slices"
+	"time"
+
+	"csh/cluesheet/config"
+	"csh/cluesheet/log"
+
+	"github.com/DataDog/dd-trace-go/v2/ddtrace/tracer"
+
+	muxtrace "github.com/DataDog/dd-trace-go/contrib/gorilla/mux/v2"
+	"github.com/gorilla/mux"
+
+	pgxtrace "github.com/DataDog/dd-trace-go/contrib/jackc/pgx.v5/v2"
+	"github.com/jackc/pgx/v5"
+
+	"github.com/google/uuid"
+
+	"go.uber.org/zap"
+)
+
+const (
+	connStr = "postgres://postgres:test@localhost:5432/postgres?sslmode=disable"
```

Comment:
Why is this hardcoded in the app? Guessing the config will override it but not how I typically do things.